lib/calculator.html

```diff
@@ -1,4 +1,5 @@
-<div class="row">
+<div class="calculator">
```

Comment:
We wrap the calculator widget HTML in a div with the `calculator` class so that it gets the calculator styles. This used to be done in the index.html file but it should really be the responsibility of calculator.html.